src/iris/api.py

```diff
@@ -3607,7 +3616,7 @@ def json_error_serializer(req, resp, exception):
     resp.content_type = 'application/json'
 
 
-def construct_falcon_api(debug, healthcheck_path, allowed_origins, iris_sender_app, sender_addr):
+def construct_falcon_api(debug, healthcheck_path, allowed_origins, iris_sender_app, zk_hosts, default_sender_addr):
```

Comment:
That file and those functions could use a refactor, and that will be addressed in a later PR.